src/adapters/MeetingsJSONAdapter.js

```diff
@@ -226,7 +226,7 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
       const meeting = this.datasource[ID];
 
       if (meeting) {
-        observer.next(meeting.localAudio.muted ? muted : unmuted);
+        observer.next(meeting.localAudio ? muted : unmuted);
```

Comment:
I like this concept of checking the object instead of diving into the stream properties. 👍